src/app/blocks/file-input-block/file-input-block.component.ts

```diff
@@ -1,6 +1,8 @@
 import {Component} from '@angular/core';
```

Comment:
There are a fair number of changes required to support this functionality. 
I have been considering adding it to the base class, or creating a new extension of the base class to make it easier to implement. 
Just not this time.